components/src/Guidelines/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react'
+import { Styled, Text as ThemeUIText } from 'theme-ui'
+import { Box } from 'theme-ui'
+
+export const Guidelines = ({ children }: ListProp) => (
+    <Box>
+        {children}
+    </Box>
+)
+
+const Dos = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
+        sx={{
+            fontFamily: 'sans serif',
+            lineHeight: 2,
+            letterSpacing: 1,
+
+        }}>
+
+        <ThemeUIText as="h3">{icon}{title}</ThemeUIText>
```

Comment:
This whole part is like a header for our section, right? Also, the only part that is actually always a text component is the title, since we receive it as a string. Therefore, you should change this into a container with two elements.

You don't need to extract it into a different component, but you should wrap them in a container. How about using `Flex` from `theme-ui`? It would look something like this:
```jsx
<Flex>
  {icon}<ThemeUIText ...>{title}</ThemeUIText>
</Flex>
```